Document/0x06f-Testing-Local-Authentication.md

```diff
@@ -61,21 +61,22 @@ In the following example we will save the string "test_strong_password" to the K
 ##### Swift
 
 ```swift
-
 // 1. create AccessControl object that will represent authentication settings
 
 var error: Unmanaged<CFError>?
 
 guard let accessControl = SecAccessControlCreateWithFlags(kCFAllocatorDefault,
-    kSecAttrAccessibleWhenPasscodeSetThisDeviceOnly,
-    .touchIDCurrentSet,
-    &error) else {
+                                                          kSecAttrAccessibleWhenPasscodeSetThisDeviceOnly,
+                                                          SecAccessControlCreateFlags.biometryCurrentSet,
```

Comment:
good find! Can you update line 59 where we reference to (`.touchIDCurrentSet parameter`)? because there we should refer to the new parameter as well for swift i guess.

Reply:
Good catch - updated!